hlx_statics/styles/styles.css

```
@@ -191,6 +191,10 @@ main .inline-code {
   line-height: 1.5rem;
 }
 
+main .header-content a .inline-code , .header-content .inline-code {
```

Please select wrapper first when possible, to follow the pattern that we have for blocks:

```suggestion
main .default-content-wrapper .header-content a .inline-code , .default-content-wrapper .header-content .inline-code {




```